HelpfulItem.cpp

```diff
@@ -3,4 +3,4 @@
 void HelpfulItem::use(Character *character)
 {
     
-}
\ No newline at end of file
+}
```

Comment:
You need to make the helpful item actually helpful.  Look to the Defensive Item or AttackItem for ideas on what to do.
- [ ] revise